.travis.yml

```diff
@@ -1,34 +0,0 @@
-language: ruby
-sudo: false
-script: 'script/cibuild'
```

Comment:
Looks like we can also remove this file.

Reply:
I couldn't find any consumers, and the file and directory don't seem to be referenced by the GitHub Actions [workflow file](https://github.com/Shopify/shipit-engine/blob/master/.github/workflows/main.yml) but I'm not 100% sure. I'll remove that in a subsequent PR, it'll be easier to revert/rollback that way.